test/unit.js

```diff
@@ -235,6 +251,14 @@ describe('skatejs-named-slots', function () {
       expect(slot.getAssignedNodes().length).to.equal(1);
     });
 
+    it('innerHTML (with [ attributes)', function () {
+      expect(host.innerHTML).to.equal('');
+      host.innerHTML = '<div [ foo ] ( bar )></div>';
```

Comment:
If you run the names together like:

```html
<div [foo] (bar)></div>
```

It may simplify the test. Not sure if you did that for a specific reason though.